README.md

```diff
@@ -1,4 +1,4 @@
-*NOTE: This README reflects the latest version of Mirador, Mirador 3. For previous versions, please reference that release's README directly. Latest 2.x release: [v.2.7.0](https://github.com/ProjectMirador/mirador/tree/v2.7.0)*
+⚠️ This README reflects the latest version of Mirador, Mirador 3. Mirador 2 is no longer actively maintained. [ProjectMirador/mirador](https://github.com/projectmirador/mirador) will no longer accept pull requests or support requests for Mirador 2. Legacy documentation can be consulted on the [Mirador 2 wiki](https://github.com/ProjectMirador/mirador-2-wiki/wiki). 
```

Comment:
```suggestion
⚠️ This project is for the Mirador 3, the latest version of Mirador. For Mirador 2, please see  [ProjectMirador/mirador2](https://github.com/projectmirador/mirador2) or legacy documentation on the [Mirador 2 wiki](https://github.com/ProjectMirador/mirador-2-wiki/wiki). Please note that the community's focus is now on Mirador 3 and the future, and is unlikely to accept pull requests or provide support for Mirador 2.
```

🤷‍♂️

Reply:
Tweaked it a bit but it looks good to me.